lib/tori/backend/s3.rb

```diff
@@ -12,9 +12,27 @@ class S3
       #
       # example:
       #   Tori.config.backend = Tori::Backend::S3.new(bucket: 'tori_bucket')
-      def initialize(bucket:)
-        @bucket = bucket
-        @client = nil
+      def initialize(options = {})
+        @bucket = options[:bucket] || (fail ArgumentError, 'missing keyword: bucket')
+
+        region =
+          options[:region] || ENV['TORI_AWS_REGION'] || ENV['AWS_REGION'] || Aws.config[:region]
+        @client =
+          case
+          when options[:access_key_id] && option[:secret_access_key]
+            Aws::S3::Client.new(
+              access_key_id:     options[:access_key_id],
+              secret_access_key: options[:secret_access_key],
+              region:            region)
+          when ENV['TORI_AWS_ACCESS_KEY_ID'] && options[:TORI_AWS_SECRET_ACCESS_KEY]
```

Comment:
`options[:TORI_AWS_SECRET_ACCESS_KEY]` looks not use.
This is `ENV['TORI_AWS_SECRET_ACCESS_KEY']` dont you ?